README.md

```diff
@@ -9,3 +9,4 @@
 # rebase 
 * adding first feature from rebase
 * developement in progress
```

Comment:
i think , it is not required , make sure it!

Reply:
its better to maintain each commit